### PR TITLE
[2.5] Fix search issues on overview page

### PR DESCRIPTION
**Pull request by zlayne**
Signed-off-by: Zack Layne <zlayne@redhat.com>

Related issue: https://github.com/stolostron/backlog/issues/23135

### CODE REVIEW

File: frontend/src/routes/Home/Overview/OverviewPage.tsx
Comment (jlpadilla) [inline frontend/src/routes/Home/Overview/OverviewPage.tsx:546]: I'd make this message always `info`.
Maybe we could do warning, but let's stay away from `danger`

**Review by jlpadilla — CHANGES_REQUESTED** [review]

**Comment by jlpadilla** [comment]
Should we merge to `main` first and then `release-2.5` ?

**Comment by zlayne** [comment]
> Should we merge to `main` first and then `release-2.5` ?

I merged another PR to main that contained a few other changes in addition to these (see: https://github.com/stolostron/console/pull/1758).

File: frontend/src/routes/Home/Overview/OverviewPage.tsx
Comment (jlpadilla) [reply frontend/src/routes/Home/Overview/OverviewPage.tsx:546]: I'm trying to avoid `danger` because from the Overview page users are interpreting it as RCHAM (the entire product) is broken.  Now that we are able to display partial data, I want to send the message that only search (single component of RHACM) is experiencing problems.  This problem reduces functionality, but it isn't catastrophic or critical to the rest of RHACM.

Comment (jlpadilla) [reply frontend/src/routes/Home/Overview/OverviewPage.tsx:546]: Maybe warning is a good compromise?

Comment (zlayne) [reply frontend/src/routes/Home/Overview/OverviewPage.tsx]: Makes sense - just pushed the change.

**Review by jlpadilla — DISMISSED** [review]
Changes look good!  2.5.1 code freeze is today so we may have to wait to merge.

**Review by jlpadilla — APPROVED** [review]
/lgtm
/hold - Need approvals to merge for 2.5.x

**Comment by jlpadilla** [comment]
Tested PR with image `2.5.1-PR1762-21011c5875d2d3ed1964dc4eadb8c0088f647715`

The Overview page now works as expected (with missing data) when the search backend is unavailable.
image

**Comment by smithbw88** [comment]
https://github.com/stolostron/backlog/issues/23202

**Pull request merged**
